test/test_sparse.py

```diff
@@ -509,8 +565,121 @@ def _test_sparse_mask_fixed(self):
         expected = self.SparseTensor(i, exp_v, torch.Size([5, 4]))
         self.assertEqual(res, expected)
 
+        expected_dense = self.ValueTensor([
+            [1, 2, 300, 4],
+            [5, 6, 100, 8],
+            [9, 10, 11, 12],
+            [13, 200, 15, 16],
+            [17, 18, 19, 400],
+        ])
+        dense._sparse_copy_(x)
+        self.assertEqual(dense, expected_dense)
+
+    def prop_sparse_mask_const(self, x, s):
```

Comment:
I don't want this to be picked up by the test runner, since it doesn't know how to generate arguments to it ;)